docs/migration-guide/overview.mdx

```diff
@@ -21,14 +21,90 @@ All breaking changes are listed below. If you encounter changes that are not exp
 
 ## Installation
 
-Payload 3.0 requires a set of auto-generated files that you will need to bring into your existing project. The easiest way of aquiring these is by initializing a new project via `create-payload-app`, then replace the provided Payload Config with your own.
+Payload 3.0 requires a set of auto-generated files that you will need to bring into your existing project. The easiest way of acquiring these is by initializing a new project via `create-payload-app`, then replace the provided Payload Config with your own.
 
 ```bash
   npx create-payload-app@beta
 ```
 
 For more details, see the [Documentation](https://payloadcms.com/docs/getting-started/installation).
 
+1. **Install new dependencies of payload, next.js and react**:
+
+    ```bash
+      pnpm i next react react-dom payload @payloadcms/ui @payloadcms/next
```

Comment:
These were likely installed already through cpa which is listed in the previous bullet

Reply:
The point of listing them is if they're updating all the versions, we pin versions now so this mattters.

The idea that they should just run CPA and then copy files to me is really incomplete as to how they should update. It also is not very clear.

Reply:
Not advocating against this, just need to be as clear as possible